rules/apps.libsonnet

```diff
@@ -100,7 +100,7 @@
           },
           // workload aggregation for deployments
           {
-            record: 'mixin_pod_workload',
+            record: 'cluster_namespace_workload_pod:mixin_pod_workload',
```

Comment:
The recording rule name should not contain cluster as that is configurable, and only valid in multi-cluster mode.